syntax = "proto3";

package streamschema;

message Data2d {
    float x = 1;
    float y = 2;
}

message Player {
    string name = 1;
    string role = 3;
    string strategy = 4;
}

message Match {
    string team = 1;
    string match = 2;
}

message Position {
    uint64 sensor_id = 1;
    uint64 timestamp_usec = 2;
    Data2d location = 3;
    Player player_info = 4;
    Match match_info = 5;
}

